docs/guides/integrating-the-feature-hub.md

```diff
@@ -161,14 +166,15 @@ integrator:
 </section>
 ```
 
-## React `FeatureAppContainer`
+### React Feature App Container
 
-The React `FeatureAppContainer` allows the integrator to bundle Feature Apps
+The `FeatureAppContainer` component allows the integrator to bundle Feature Apps
 instead of loading them from a remote location.
```

Comment:
Not necessarily part for this PR but we should explain when this is and isn't a good idea...